lib/rubycritic/cli/application.rb

```diff
@@ -9,17 +9,24 @@ class Application
       STATUS_ERROR   = 1
 
       def initialize(argv)
+        puts argv.class
```

Comment:
A wiser man than me once gave me a tip I still follow up until this day: Never indent your debugging statements. That way they'll always stick out like a sore thumb ;)